--- KGtk-0.9.4/gtk2/kgtk2-wrapper.cmake~	2007-10-31 21:33:19.000000000 +0100
+++ KGtk-0.9.4/gtk2/kgtk2-wrapper.cmake	2008-03-02 03:32:20.000000000 +0100
@@ -15,7 +15,7 @@
 app=`basename $0`
 
 if [ "$app" = "kgtk2-wrapper" ] ; then
-    LD_PRELOAD=@CMAKE_INSTALL_PREFIX@/lib/kgtk/libkgtk2.so:$LD_PRELOAD "$@"
+    LD_PRELOAD=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@/kgtk/libkgtk2.so:$LD_PRELOAD "$@"
 else
     dir=`dirname $0`
     oldPath=$PATH
